pkg/controlplane/controller.go

```diff
@@ -110,11 +112,11 @@ func (c *completedConfig) NewBootstrapController(legacyRESTStorage corerest.Lega
 		SecondaryServiceClusterIPRegistry: legacyRESTStorage.SecondaryServiceClusterIPAllocator,
 		SecondaryServiceClusterIPRange:    c.ExtraConfig.SecondaryServiceIPRange,
 
-		ServiceClusterIPInterval: 3 * time.Minute,
+		ServiceClusterIPInterval: c.RepairServicesInterval,
```

Comment:
@liggitt is this allowed?
I need to expose this option to the test integration framework, otherwise the test has to last 3 mins

Reply:
pretty sure we already have something similar for endpoint reconciler configuration (in ExtraConfig, not bound to a flag that I can see)